modules/op-entitlements-manual-copying.adoc

```diff
@@ -0,0 +1,78 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/configuring-security-context-for-pods.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="op-entitlements-manual-copying_{context}"]
+= Using Red Hat entitlements by manually copying the etc-pki-entitlement secret
+
+You can copy the `etc-pki-entitlement` secret from the `openshift-config-managed` namespace into the namespace of your pipeline. You can then configure your pipeline to use this secret for the Buildah task.
+
+.Prerequisites
+
+* You installed the `jq` package on your system. This package is available in {op-system-base-full}.
+
+.Procedure
+
+. Copy the `etc-pki-entitlement` secret from the `openshift-config-managed` namespace into the namespace of your pipeline by running the following command:
++
+[source,terminal]
+----
+$ oc get secret etc-pki-entitlement -n openshift-config-managed -o json | \
+  jq 'del(.metadata.resourceVersion)' | jq 'del(.metadata.creationTimestamp)' | \
+  jq 'del(.metadata.uid)' | jq 'del(.metadata.namespace)' | \
+  oc -n _namespace_ create -f - <1>
```

Comment:
It renders it as `_namespace_` (with underscores) which probably wasn't intention.